README.md

```diff
@@ -155,6 +155,39 @@ if err != nil {
 }
 ```
 
+### CLI to replay dry-run files
+
+A small helper binary is included to ingest JSON files created by the
+`DryRunClient` and send them to a running Diode service.
+
+Install the helper using `go install`:
+
+```bash
+go install github.com/netboxlabs/diode-sdk-go/cmd/diode-replay-dryrun@latest
+```
+
+This installs the command separately from the SDK library obtained with
+`go get`.
+
+Run it by providing one or more JSON files and connection details. Use `-file`
+multiple times to ingest several dry-run files in a single request:
+
+```bash
+diode-replay-dryrun \
+  --file /tmp/example-app_1750106879725947344.json \
+  --file /tmp/other.json \
+  --target grpc://localhost:8080/diode \
+  --app-name example-app \
+  --app-version 0.1.0 \
+  -client-id YOUR_CLIENT_ID \
+  -client-secret YOUR_CLIENT_SECRET
```

Comment:
There's inconsistency about flags usage in this example and ones mentioned in L186-L189, shouldn't it be as below?
```suggestion
diode-replay-dryrun \
  -file /tmp/example-app_1750106879725947344.json \
  -file /tmp/other.json \
  -target grpc://localhost:8080/diode \
  -app-name example-app \
  -app-version 0.1.0 \
  -client-id YOUR_CLIENT_ID \
  -client-secret YOUR_CLIENT_SECRET
```